src/agent/rustjail/src/seccomp.rs

```diff
@@ -39,6 +39,20 @@ fn get_rule_conditions(args: &[LinuxSeccompArg]) -> Result<Vec<ScmpArgCompare>>
     Ok(conditions)
 }
 
+pub fn check_unknown_syscalls(scmp: &LinuxSeccomp) -> String {
```

Comment:
It would be more idiomatic to make this `-> Result<Vec<String>>` or similar.

Reply:
I use `Option<Vec<String>` instead of `Result<Vec<String>>`.